### PR TITLE
adding support for meta tag name attribute

### PR DESCRIPTION
**Pull request by megantaylor**
to resolve https://github.com/ospfranco/link-preview-js/issues/68

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -75,7 +82,7 @@ function getImages(doc: any, rootUrl: string, imagesPropertyType?: string) {
     } else {
       nodes = doc(`img`);
 
-      if (nodes.length) {
+      if (nodes) {
```
Comment (ospfranco) [inline index.ts:85]: is this really correct? since the array can be empty, maybe `if (nodes?.length) {` is a bit more robust?

File: index.ts [hunk 2]
```diff
@@ -247,14 +261,14 @@ function parseUnknownResponse(
   body: string,
   url: string,
   options: ILinkPreviewOptions = {},
-  contentType?: string,
```
Comment (ospfranco) [inline index.ts:250]: Are you using the same eslint rule set?

File: index.ts (hunk 1 above)
Comment (megantaylor) [reply index.ts:85]: yeah, good point, will fix

File: index.ts (hunk 2 above)
Comment (megantaylor) [reply index.ts:250]: i thought i was, but maybe not...

Comment (megantaylor) [reply index.ts:250]: fixing now

File: index.ts (hunk 1 above)
Comment (megantaylor) [reply index.ts]: fixed

File: index.ts (hunk 2 above)
Comment (megantaylor) [reply index.ts]: fixed

File: index.ts [hunk 3]
```diff
@@ -108,16 +102,15 @@ function getVideos(doc: any) {
...
-  const nodes = doc(`meta[property='og:video']`);
-  const { length } = nodes;
+  const nodes = metaTag(doc, `og:video`, `property`) || metaTag(doc, `og:video`, `name`);
 
-  if (length) {
-    nodeTypes = doc(`meta[property='og:video:type']`);
-    nodeSecureUrls = doc(`meta[property='og:video:secure_url']`);
-    width = doc(`meta[property='og:video:width']`).attr(`content`);
-    height = doc(`meta[property='og:video:height']`).attr(`content`);
+  if (nodes) {
```
Comment (ospfranco) [inline index.ts:107]: also apply: `nodes?.length` here and we should be good to go

Comment (megantaylor) [reply index.ts]: done!

**Review by ospfranco — APPROVED** [review]

**Comment by ospfranco** [comment]
Cheers @megantaylor

**Pull request merged**